etc/kayobe/stackhpc-monitoring.yml

```
@@ -15,3 +15,13 @@ alertmanager_low_memory_threshold_gib: 5
 # Enabling this flag will result in HAProxy configuration and Prometheus scrape
 # targets being templated during deployment.
 stackhpc_enable_os_capacity: false
+
+# Keystone authentication URL for OpenStack Capacity
+stackhpc_os_capacity_auth_url: "http{% if kolla_enable_tls_internal %}s{% endif %}://{{ internal_net_vip_address }}:5000"
```

internal_net is a specific network that is not guaranteed to be defined. You'd need to use the generic `internal_net_name | net_vip_address` formulation. But it will be easier to use `kolla_internal_fqdn`. In general it's best to use the FQDN rather than the VIP address, since this is what the certificate will be valid for.

```suggestion
stackhpc_os_capacity_auth_url: "http{% if kolla_enable_tls_internal | bool %}s{% endif %}://{{ kolla_internal_fqdn }}:5000"
```